src/lib/api/EventAPI.ts

```diff
@@ -177,6 +179,44 @@ export const deletePickupEvent = async (token: string, pickupEvent: UUID): Promi
   });
 };
 
+export const completePickupEvent = async (
+  token: string,
+  uuid: UUID
+): Promise<CompleteOrderPickupEventResponse> => {
+  const requestUrl = `${config.api.baseUrl}${config.api.endpoints.store.pickup.single}/${uuid}/complete`;
+
+  const response = await axios.post(
+    requestUrl,
+    {},
+    {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    }
+  );
+
+  return response.data.orders;
+};
+
+export const cancelPickupEvent = async (
+  token: string,
+  uuid: UUID
+): Promise<CancelOrderPickupEventResponse> => {
```

Comment:
since it looks like `CancelOrderPickupEventResponse` doesn't return anything meaningful, you can just return `Promise<void>` instead